modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1545,6 +1545,8 @@ public ObjectEntry updateObjectEntry(
 
 		objectEntryPersistence.clearCache(SetUtil.fromArray(objectEntryId));
 
+		ObjectEntry originalObjectEntry = objectEntry;
+
 		objectEntry = objectEntryPersistence.findByPrimaryKey(objectEntryId);
```

Comment:
@G-Sa we are calling `objectEntryPersistence.findByPrimaryKey` at the beginning of this method ([here](https://github.com/liferay-objects/liferay-portal/blob/8be3be73f0095c7256a43ca42e0eb16bc98a7181/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java#L1512)). Why did you decide to save the original only at this point in the code?

Reply:
I just went as far down as i could simply to be nearest to the point where the variable will be used

Reply:
@G-Sa but is it possible that before this code the `objectEntry` was already updated? I did not understand why we are calling the `findByPrimaryKey` method twice.

Can you check how it worked before (when using the model listener)?